.travis.yml

```diff
@@ -122,8 +122,19 @@ matrix:
         - NUMPYSPEC="numpy==1.17.4"
         - USE_SDIST=1
     - python: 3.6
-      if: type = pull_request
+      name: "Wheel, Optimised, GCC 4.8"
```

Comment:
the test incantation command is different on the older branch it seems.. since it included a COVERAGE entry already I left it alone, but not sure what will happen..